src/osp/Resource/Package.h

```diff
@@ -61,12 +61,26 @@ namespace osp
 
 // supported resources
 
+// Just a string, but typedef'd to indicate that it represents a prefix
+typedef std::string resprefix_t;
```

Comment:
I may be misremembering, but I thought the project's coding guidelines document encouraged capitalizing the first letter in the typedef name?